tutorials/VI_in_BM.ipynb

```diff
@@ -0,0 +1,2009 @@
+{
```

Comment:
<p>Can we ad a snippet to install Bean Machine if it's not found, similar to <a href="https://github.com/facebookresearch/beanmachine/blob/main/tutorials/Coin_flipping.ipynb" target="_blank">what we do in the other tutorials</a>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/facebookresearch/beanmachine/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>